buildscripts/winbuild.sh

```diff
@@ -67,6 +67,10 @@ function install_python(){
 		echo "Installing vc_redist (2008) for 32 bit "
 		wine vcredist_x86.exe /Q
 	fi
+	# add cert
+	if [ -f /usr/local/share/ca-certificates/bitmessage-proxy.crt ]; then
+		wine python -m pip config set global.cert 'z:\usr\local\share\ca-certificates\bitmessage-proxy.crt'
```

Comment:
I'm afraid that `z:\usr\local\share\ca-certificates\bitmessage-proxy.crt` doesn't translates to `/usr/local/share/ca-certificates/bitmessage-proxy.crt`, but rather in something like `$HOME/.wine64/dosdevices/z\:usr\local\share\ca-certificates\bitmessage-proxy.crt`

Reply:
Though, it may be the same locations.

Reply:
it works correctly for me